projects/cashmere-examples/src/lib/form-field-overview/form-field-overview-example.html

```diff
@@ -0,0 +1,29 @@
+<form>
+    <div class="form-sample">
+        <hc-form-field>
+            <hc-label>Item Name:</hc-label>
+            <input hcInput required [formControl]="inputControl">
+            <hc-error>An item name is required</hc-error>
```

Comment:
This is more of a question than a suggestion, but is there a way to handle different types of errors? 

What if I wanted to validate required OR max length?

Reply:
Yup, I think we're in good shape there.  Jeremy had everything set up so you can use [Validators](https://angular.io/api/forms/Validators) on [FormControl](https://angular.io/api/forms/FormControl) which should cover just about any type of errors you'd want to check for.  Take a look at the example `input-required` to see that being used.